api/specs/web-server/_catalog.py

```diff
@@ -34,6 +36,33 @@
 #
 
 
+@router.get(
+    "/dev/catalog/services/-/latest",
+    response_model=Envelope[list[DEVServiceGet]],
+)
+def dev_list_services_latest(_query_params: Annotated[ListServiceParams, Depends()]):
+    pass
+
+
+@router.get(
+    "/dev/catalog/services/{service_key}/{service_version}",
+    response_model=Envelope[DEVServiceGet],
+)
+def dev_get_service(_path_params: Annotated[ServicePathParams, Depends()]):
```

Comment:
I am wondering a bit about this decision:
- why do you create new entries for this? they are essentially exactly the same as the old ones.
- Is the plan to go back to the old ones afterwards?
- services/-/latest == services?filter=latest

Reply:
- yes, they will replace the old when we integrate them in the front-end. This is a draft
- when these are finished and integrated in the front-end we will remove the `/dev` prefix and drop the old ones
- yes. The idea is `services/{key}/{version}` and therefore `services/-/latest` refers to the latest of all versions. A filter would also be an option but the thing is that @odeimaiz  only needs listing `latest` and I prefer to "hard-code" it instead of giving the impression that we can filter any version (which would require more coding).